### ISSUE TITLE
PRO bug. removeControlSpinner() not working

### ISSUE THREAD

**Issue by collizo4sky** [elementor/elementor#8790]
On elementor-pro/assets/js/editor.js line 197, `elementor-control-spinner` is missing a leading dot. 

This

```
removeControlSpinner: function removeControlSpinner(name) {
		var $controlEl = this.getEditorControlView(name).$el;

		$controlEl.find(':input').attr('disabled', false);
		$controlEl.find('elementor-control-spinner').remove();
	},
```

Should be

```
removeControlSpinner: function removeControlSpinner(name) {
		var $controlEl = this.getEditorControlView(name).$el;

		$controlEl.find(':input').attr('disabled', false);
		$controlEl.find('.elementor-control-spinner').remove();
	},
```

**Comment by bainternet** [comment]
thanks.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.7

Feel free to update

Thanks!
